src/sql/sqlops.proposed.d.ts

```diff
@@ -411,6 +421,16 @@ declare module 'sqlops' {
 				 * A string giving the ID of the tab's model view content
 				 */
 				content: string;
+
+				/**
+				 * Returns he model view content
+				 */
+				modelView: ModelView;
+
+				/**
+				 * Register model view content for the dialog
+				 */
+				registerContent(handler: (view: ModelView) => void): void;
```

Comment:
minor: why not define a shared interface and have Dialog and DialogTab inherit from it so the methods are the same, and any additional API that needs this just inherits it too.